python/lsst/the/monster/assemble.py

```diff
@@ -37,44 +39,75 @@ class AssembleMonsterRefcat:
     photometry from the DES-calibrated catalogs. The catalogs are read
     in reverse priority order so that the final fluxes for any object
     come from the top priority catalog. At the end the intermediate
-    calibrated catalogs are converted to the final fluxes (e.g. LSST).
+    calibrated catalogs are converted to the final fluxes in the target
+    systems specified in target_catalog_info_class_list.
 
     Parameters
     ----------
     gaia_reference_class : `RefcatInfo`
         The input Gaia DR3 RefcatInfo object.
     catalog_info_class_list : `list` [`RefcatInfo`]
         Reverse-priority list of catalog info classes for assembly.
+    target_catalog_info_class_list : `list` [`RefcatInfo`]
+        List of catalog info classes that will be output in final monster.
     monster_path_inp : `str`, optional
         Output monster path, overriding the class config.
+    do_u_band_slr : `bool`, optional
+        Perform u-band SLR using SDSS u and DES g & r bands
```

Comment:
This isn't quite right.  The SLR goes from DES g/g-r to something like SDSS u.  Maybe it's just a wording issue.